app/ide-desktop/lib/dashboard/src/index.html

```diff
@@ -1 +1 @@
-../../content/src/index.html
\ No newline at end of file
+../../content/src/index.html
```

Comment:
uh oh... i can't tell what happened here from the github ui, but chances are it's the classic "windows turned the symlink into a not symlink"

Reply:
Think this is fixed now

Reply:
weird... seems like prettier thinks that file is HTML and overwrites it, you might have to manually `git checkout develop app/ide-desktop/lib/dashboard/src/index.html`